src/components/CallView/BottomBar.vue

```diff
@@ -104,6 +112,59 @@ const showCallLayoutSwitch = computed(() => !callViewStore.isEmptyCallView)
 const isGrid = computed(() => callViewStore.isGrid)
 const userIsInBreakoutRoomAndInCall = computed(() => conversation.value.objectType === CONVERSATION.OBJECT_TYPE.BREAKOUT_ROOM)
 
+const COLLAPSIBLE_BUTTONS = ['virtualBackground', 'liveTranscription', 'raiseHand', 'callLayout', 'fullscreen'] as const
+const isActionAvailableMask = computed(() => ({
+	fullscreen: !isSidebar,
+	callLayout: showCallLayoutSwitch.value,
+	raiseHand: true,
+	liveTranscription: isLiveTranscriptionSupported.value,
+	virtualBackground: !isSidebar,
+}))
+const hidingList = ref(Object.fromEntries(COLLAPSIBLE_BUTTONS.filter((item) => isActionAvailableMask.value[item]).map((item) => [item, false])))
```

Comment:
Same, and we ensure later change in computed doesn't affect structure of the ref
```suggestion
const hidingList = ref<CollapsibleButtons>({ ...isActionAvailableMask.value })
```